website_form_builder/static/src/js/snippets.js

```diff
@@ -222,13 +222,11 @@ odoo.define('website_form_builder.snippets', function (require) {
             if (fields.length) {
                 // Whitelist model fields found in current form
                 this._rpc({
-                    model: 'ir.model.fields',
-                    method: 'formbuilder_whitelist',
+                    model: "ir.model.fields",
+                    method: "formbuilder_whitelist",
                     args: [this.controller_data().model_name, fields],
-                    kwargs: {
-                        context: weContext.get(),
-                    },
-                    async: false
+                    kwargs: {context: weContext.get()},
+                    async: false,
```

Comment:
```suggestion
                    "async": false,
```

This is a reserved word in modern JS.